test/integration/ddl_sharding_key_test.lua

```diff
@@ -0,0 +1,370 @@
+local fio = require('fio')
+
+local t = require('luatest')
+local crud = require('crud')
+
+local helpers = require('test.helper')
+
+local ok = pcall(require, 'ddl')
+if not ok then
+    t.skip('Lua module ddl is required to run test')
+end
+
+local pgroup = helpers.pgroup.new('ddl_sharding_key', {
+    engine = {'memtx', 'vinyl'},
+})
+
+pgroup:set_before_all(function(g)
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        server_command = helpers.entrypoint('srv_ddl'),
+        use_vshard = true,
+        replicasets = helpers.get_test_replicasets(),
+        env = {
+            ['ENGINE'] = g.params.engine,
+        },
+    })
+    g.cluster:start()
+    local result, err = g.cluster.main_server.net_box:eval([[
+        local ddl = require('ddl')
+
+        local ok, err = ddl.get_schema()
+        return ok, err
+    ]])
+    t.assert_equals(type(result), 'table')
+    t.assert_equals(err, nil)
+
+    g.space_format = g.cluster.servers[2].net_box.space.customers:format()
+end)
+
+pgroup:set_after_all(function(g) helpers.stop_cluster(g.cluster) end)
+
+pgroup:set_before_each(function(g)
+    helpers.truncate_space_on_cluster(g.cluster, 'customers')
+    helpers.truncate_space_on_cluster(g.cluster, 'writers')
+end)
+
+local function check_get(g, space_name, id, name)
+    local result, err = g.cluster.main_server.net_box:call('crud.get', {
+        space_name, {id, name}
+    })
+
+    t.assert_equals(err, nil)
+    t.assert(result ~= nil)
```

Comment:
Why not `t.assert_not_equals(result, nil)`?

Reply:
Sure. Fixed.